filebeat/module/nginx/ingress_controller/ingest/pipeline.yml

```
@@ -41,6 +61,81 @@ processors:
     field: source.address
     if: ctx.source?.address == null
     value: ""
+- script:
+    if: ctx.nginx?.ingress_controller?.upstream?.response?.length_list != null && ctx.nginx.ingress_controller.upstream.response.length_list.length > 0
+    lang: painless
+    source: >-
+      try {
+        if (ctx.nginx.ingress_controller.upstream.response.length_list.length == null) {
+          return;
+        }
+        int length = 0;
+        for (def item : ctx.nginx.ingress_controller.upstream.response.length_list) {
+          length = length + Integer.parseInt(item);
+        }
+        ctx.nginx.ingress_controller.upstream.response.length = length;
```

Maybe change the name to indicate it's a total value?
```suggestion
        ctx.nginx.ingress_controller.upstream.response.length_total = length;
```

---

Take into account that this is a breaking change as it changes an existing field. I would prefer to keep the original field.

---

👍